src/routes/reference/melbands.svx

```diff
@@ -1,8 +1,86 @@
 ---
 title: MelBands
-blurb: Amplitude for a number of equally spread perceptual bands
+blurb: Magnitudes for a number of perceptually-evenly spaced spectral bands
 tags: 
     - descriptor
+    - mfcc
+    - mel
+    - perception
+    - spectra
 flair: reference
 category: Analyse Sound
----
\ No newline at end of file
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+    import YouTube from '$lib/components/YouTube.svelte';
+    import { Tabs, TabList, TabPanel, Tab } from '$lib/components/tabs/tabs';
+</script>
+
+`MelBands` returns a Mel-Frequency Spectrum which is a [histogram](https://en.wikipedia.org/wiki/Histogram) of FFT bin magnitudes bundled according to their relationship to the [Mel scale](https://en.wikipedia.org/wiki/Mel_scale). The Mel scale represents frequency space logarithmically, mimicking how humans perceive distance in pitch space. (The name "Mel" derives from the word "melody".)
+
+<YouTube
+url='UCLlVAj0PPY'
+/>
+
+In order to create the Mel-Frequency Spectrum, first an FFT is computed on a window of the signal. Next, a series of overlapping triangle filters are applied to the magnitudes of the FFT spectrum: each magnitude in the FFT spectrum is multiplied by its corresponding value in the each triangle filter. These products are then added up to produce a weighted sum of FFT magnitudes for each triangle filter, which is the magnitude of the corresponding MelBand. Since most of the values in these triangle filters are zero (everywhere except where the triangle rises up) each filter only sums the magnitudes in a certain frequency band, so each MelBand magnitude is a representation of the overall energy in that frequency band. The MelBands are perceived as being equally spaced because as they get higher in frequency, they cover more FFT bins (more hertz), mirroring how humans perceive pitch relationships (higher frequency pitches have more hertz between musical intervals).
+
+One can choose whether to `normalize` triangle filters used to compute the MelBands. By default (unnormalized, `normalize` = 0), the triangle filters account for how many FFT magnitudes they sum together by dividing the sum by the number of contributing FFT magnitudes, essentially averaging their contributions. When the triangle filters _are_ normalized (`normalize` = 1), averaging does not occur and the magnitudes of the higher MelBands end up being larger values (disproportionately so) becuase they're summing together more FFT magnitudes. In the charts below, notice how different the unnormalized and normalized triangle filters are and when normalized, how much larger the magnitudes in the higher MelBands become.
```

Comment:
normalization is *on* by default

Reply:
I think the explanation is backwards too. When normalization is on, the bands are scaled proportional to the linear area they occupy, so the higher bands are turned down